### DISCUSSION TITLE
Having some issues with launching Jupyter : 403 Forbidden Error

### DISCUSSION THREAD

**Discussion by nebabebe** [cooperrc/computational-mechanics#242] [Q&A] [answered]
Hi everyone! 

Has anyone came across this 403 : Forbidden error yet? I was attempting to clone my forked repository into my Jupyter server, but after I logged in with my UConn credentials. This is what showed up. (I've attached a screenshot) 

Any tips or tricks to get around this?

I've seen a previous post where my UConn net ID has to be on a user list, if that is needed, my net ID is _____

Thanks in advance!

![image](https://github.com/cooperrc/computational-mechanics/assets/171044627/df77101c-85e3-4599-83a1-e27c9fe0f21b)

**Comment by cooperrc** [comment]
Use the nanohub compmech server. 

https://nanohub.org/tools/compmech/

**Reply by cooperrc** [reply]
It has all of the course notebooks included already, then if you want to keep your work on github we can work on updating your fork with the nanohub notebook work

**Reply by nebabebe** [reply]
Thank you!

**Comment by LucyLyttle** [comment]
I am having a similar problem but with a different error code. When I try to select "launch tool" and sign in on Nanohub I get a 503 error and this message, and I haven't been able to get around it. I'm not sure what to do. I've tried several times today and it hasn't worked.

![image](https://github.com/cooperrc/computational-mechanics/assets/171046286/4dd6fc2a-e858-43bd-8ea0-c4d1e9bdea75)

**Reply by nebabebe** [reply, marked as answer]
Hey!

I found a solution! It's actually because we are trying to create an account using the drop down menu. Instead of using the drop down menu and typing in "University of Connecticut", create a new log in by just manually signing up. After I did this, I was able to gain access right into Jupyter with no issues. 

Start here: https://nanohub.org/tools/compmech/

Click "Launch Tool" 

Then click "Create an Account" 

I've included a screenshot on where to click so that it's a bit more clear!

![Screenshot 2024-05-30 191300](https://github.com/cooperrc/computational-mechanics/assets/171044627/98bddb9a-ff9b-42a7-b3c1-fd5d5402eedc)

**Reply by LucyLyttle** [reply]
Thank you! That worked!
